spec/contracts/work_packages/base_contract_spec.rb

```diff
@@ -476,6 +476,72 @@
     end
   end
 
+  describe 'duration' do
+    context 'when setting the duration' do
+      before do
+        work_package.duration = 5
+      end
+
+      it_behaves_like 'contract is valid'
+    end
+
+    context 'when setting the duration to 0' do
```

Comment:
There is a missing test for this case:

> Duration is always in full days (never floating point values).

Reply:
Fixed in f5fc2b9 which turned out to be harder than expected because the `only_integer:` validation is carried out on the already type casted value.

Reply:
The translation is missing. I have `translation missing: en.activerecord.errors.models.work_package.attributes.duration.only_integer` message.

Reply:
according to https://guides.rubyonrails.org/i18n.html#error-message-interpolation, the right i18n key is `:not_an_integer`. I'll fix it.

Reply:
Done in 90a8329c3a